articles/azure-netapp-files/azure-netapp-files-quickstart-set-up-account-create-volumes.md

```diff
@@ -100,9 +100,9 @@ You need to be part of the Public Preview program and whitelisted for accessing
    4. Use the default value for quota. 
    5. Under virtual network, click **Create new** to create a new Azure virtual network (Vnet).  Then fill in the following information:
        * Enter **myvnet1** as the Vnet name.
-       * Specify an address space, for example, 10.7.0.0/16.
+       * Specify an address space, for example, 10.7.0.0/16. The address space is up to you
```

Comment:
How about:
* Specify an address space for your setting, for example, 10.7.0.0/16.

(This being a Quickstart, we were told to be as explicit as possible.)